portfolio/src/main/java/com/google/sps/servlets/MapServlet.java

```diff
@@ -40,32 +40,52 @@
   private static final String DESCRIPTION = "description";
   private static final String TABLE_NAME = "Pins";
   private static final String TIME_STAMP = "timestamp";
+  private static final String ID = "id";
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    UserService userService = UserServiceFactory.getUserService();
+    Query query = new Query(TABLE_NAME);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    List<UserComment> userCommentsList = new ArrayList<UserComment>();
+    for (Entity entity : results.asIterable()) {
+      String name = (String) entity.getProperty(NAME);
+      String phone = (String) entity.getProperty(PHONE);
+      String location = (String) entity.getProperty(LOCATION);
+      String description = (String) entity.getProperty(DESCRIPTION);
+      long timeStamp = (long) entity.getProperty(TIME_STAMP);
+      long id = entity.getKey().getId();
+
+      UserComment userComment = new UserComment(name, phone, location, description, timeStamp, id);
+      userCommentsList.add(userComment);
+    }
+    System.out.println(userCommentsList);
```

Comment:
Same here, prefer logging instead of print.

Reply:
Deleted.